posthog/models/error_tracking/error_tracking.py

```diff
@@ -58,3 +58,50 @@ def convert_fingerprints_to_tuples(fps: list[list[str]]):
         # converting back to list of lists before saving
         self.merged_fingerprints = [list(f) for f in merged_fingerprints]
         self.save()
+
+
+class ErrorTrackingSymbolSet(UUIDModel):
+    class StorageMethod(models.TextChoices):
+        S3 = "s3", "s3"
+        FILE = "file", "file"
+
+    # Derived from the symbol set reference
+    ref = models.TextField(null=False, blank=False)
+    team = models.ForeignKey("Team", on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    # How we stored this symbol set, and where to look for it
+    # These are null if we failed to find a symbol set for a given reference. We store a
+    # row anyway, so if someone comes along later and uploads a symbol set for this reference,
+    # we can know which frame resolution results below to drop.
+    storage_method = models.TextField(choices=StorageMethod.choices, default=StorageMethod.S3, null=True)
+    storage_ptr = models.TextField(null=True, blank=False)
+
+    class Meta:
+        indexes = [
+            models.Index(fields=["team_id", "id"]),
+        ]
+
+        constraints = [
+            models.UniqueConstraint(fields=["team_id", "id"], name="unique_id_per_team"),
+        ]
```

Comment:
We want these to be unique on `ref` not `id`, right?

Reply:
Yes, correct